src/pages/task/Metadata.tsx

```diff
@@ -78,6 +79,7 @@ export const Metadata: React.FC<{
           </StyledLink>
         </P2>
       )}
+      {ami && <P2 data-cy="task-metadata-ami">AMI: {ami}</P2>}
```

Comment:
I think just displaying AMI isn't very useful without context. This should probably be the distro instead with a link to the distro page, and the AMI as subtext afterwards

Reply:
I added the distro link but Would a link to the Distro be anymore useful?  I believe the linked to Distro would always display the latest ami associated with the distro. So if the AMI changes for a distro it wont be reflected on the Distro page.

Reply:
I think the distro is the main piece of information since it answers questions like "what sort of machine did my task run on." The AMI is mostly a supplement to that in case the AMI has changed and the user is trying to exactly reproduce their task